site/docs/pages/transaction/transaction.mdx

```diff
@@ -123,6 +123,63 @@ export default function TransactionComponents() {
   </TransactionWrapper>
 </App>
 
+## `Transaction` with Paymaster Gas Sponsorship Capabilities
+##### Obtain a Paymaster and Bundler endpoint from the [Coinbase Developer Platform Paymaster](https://portal.cdp.coinbase.com/products/bundler-and-paymaster).
+
+<img
+  alt="OnchainKit Paymaster and Bundler endpoint"
+  title="OnchainKit Paymaster and Bundler endpoint"
+  src="/assets/onchainkit-components-paymaster-endpoint.png"
+  width="auto"
+/>
+```tsx
+// ommited for brevity
+<Transaction
+  address={address}
+  contracts={contracts}
+  capabilities={{ // [!code focus]
+    paymasterService: { // [!code focus]
+      url: process.env.PAYMASTER_AND_BUNDLER_ENDPOINT, // [!code focus]
```

Comment:
```
process.env.PAYMASTER_AND_BUNDLER_ENDPOINT
```
is this simple enough to understand? I believe we approach this in a different way in the Getting Started, or no?

Reply:
In the getting started guide we had an additional step that showed how to add the environment variable to a .env file